crates/core_arch/src/x86/sse.rs

```diff
@@ -3437,7 +3437,7 @@ mod tests {
     }
 
     #[simd_test(enable = "sse")]
-    pub unsafe fn test_mm_cvtsi32_ss() {
+    unsafe fn test_mm_cvtsi32_ss() {
```

Comment:
These tests were `pub` items for some reason and the check errored that they also needed tests..